src/resources/packages/wizard/components/tabs/organizer/tab.tsx

```diff
@@ -51,7 +51,7 @@ const OrganizerContent = ({moveToNextTab, skipToNextTab}) => {
 	}, []);
 
 	const toggleClasses = (field, fieldEle, parentEle, isValid) => {
-		if ( !field ) {
+		if ( !field && fieldEle.id === 'organizer-name' ) {
```

Comment:
Those conditionals dont make sense to me ... could you explain ?

Reply:
Note: what Dimi is getting at is the first condition is "there is no field, or it's empty" so the second one will never return true (and could throw errors/warnings).